css/main.css

```diff
@@ -27,7 +27,9 @@ header {
   color: white;
 }
 
-#desktop-nav, #desktop-nav ul {
+#desktop-nav,
+#mobile-nav,
```

Comment:
Your mobile-nav design is not the same with the figma design; it was not centered and the close button is on the left.  After going through your css file, all mobile-nav selectors were grouped with other selectors. It would be nice if you single out and style the mobile-nav selector with the required css properties to make it look like the figma design :slightly_smiling_face: